opendevin/core/config.py

```diff
@@ -173,6 +173,10 @@ class AppConfig(metaclass=Singleton):
     disable_color: bool = False
     sandbox_user_id: int = os.getuid() if hasattr(os, 'getuid') else 1000
     sandbox_timeout: int = 120
+    persist_session: bool = False
+    ssh_port: int | None = None
+    ssh_password: str | None = None
+    container_id: str | None = None
```

Comment:
we should probably factor out a SandboxConfig

Reply:
Definitely. It doesn't have to be this PR though IMHO.

Reply:
It should be done at that time. Now one needs to update the configs.